tests/PathMappingRepositoryTest.php

```diff
@@ -221,13 +221,13 @@ public function testResolveFilesystemResource()
         $resource = $repository->get('/webmozart/foo/sub');
 
         $this->assertInstanceOf('Puli\Repository\Api\Resource\FilesystemResource', $resource);
-        $this->assertEquals(__DIR__.'/Fixtures/dir5/sub', $resource->getFilesystemPath());
+        $this->assertEquals(str_replace(__DIR__, DIRECTORY_SEPARATOR, '/').'/Fixtures/dir5/sub', $resource->getFilesystemPath());
```

Comment:
I think it's fine here. On a side note, for such cases (without `.`, `..` etc.) you should use `Path::normalize()` which is more efficient.